site/_posts/2016-10-28-sprint-48-review-october-26-2016.html.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Sprint 48 review - October 26, 2016
+author: cybette
+date: 2016-10-28 20:20:00 UTC
+tags: sprints
+comments: true
+published: true
+---
+
+Welcome to the Sprint 48 review recap! This was the sprint for the [Euwe Beta](http://manageiq.org/blog/2016/10/Announcing-Euwe-Beta-Release/) release, with the just announced [Euwe release candidate](http://manageiq.org/blog/2016/10/Announcing-Euwe-Release-Candidate/) hot on its heels.
+
+In this sprint we had 404 pull requests merged in the main [ManageIQ repo](https://github.com/ManageIQ/manageiq), a big chunk consisting of bug fixes. 266 PRs were backported to the Euwe branch that we are focusing on stabilizing, and this will continue to the next sprint. Across the repos, with plenty of work going into the [new](http://manageiq.org/) [website](https://github.com/ManageIQ/manageiq.org), a lot of clean-up and new features in <del>Self</del> [Service UI](https://github.com/ManageIQ/manageiq-ui-service), as well as a [new repo to podify](https://github.com/ManageIQ/manageiq-pods) the ManageIQ project for Kubernetes/Openshift, we accomplished a total of 693 PRs in 3 weeks!
```

Comment:
When you do the `[new](http://manageiq.org/) [website](https://github.com/ManageIQ/manageiq.org)` style of linking, it can appears as a single link to users, and they may not know there are two separate clicks there.  Maybe instead something like the following?  It's relatively minor, so not sure it really needs changing...it's up to you.

`[new repo](http://manageiq.org/) [(source)](https://github.com/ManageIQ/manageiq.org)`
[new repo](http://manageiq.org/) [(source)](https://github.com/ManageIQ/manageiq.org)